terraform/instances.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_instance" "coreos-tf-example" {
```

Comment:
can we remove `-example` from the instance name to make it consistent with other resources.